lib/plugins/stonith/external/drac5.in

```diff
@@ -6,12 +6,24 @@
 # License:      GNU General Public License (GPL)
 #
 
-trap 'if [ -n "$outf" ]; then ha_log.sh err "`cat $outf`"; rm -f "$outf"; fi' 0
```

Comment:
Yes, this is obviously wrong. One possible fix would be [ -s "$outf" ]. And "rm -f" should be outside of the if statement.

Reply:
if a temp file was successfully created (filename length is non-zero), we want to remove it.
So, the rm -f is properly placed inside the if.

According to
https://downloads.dell.com/manuals/all-products/esuprt_software_int/esuprt_software_ent_systems_mgmt/idrac9-lifecycle-controller-v4x-series_white-papers1_en-us.pdf
it appears that non-zero exit codes are errors.

So, I'll include changes to check the return code and log appropriately